sway-ir/src/constant.rs

```diff
@@ -7,6 +7,12 @@ use crate::{
     value::Value,
 };
 
+/// A [`Type`] and constant value, including [`ConstantValue::Undef`] for uninitialized constants.
```

Comment:
```suggestion
/// A boot time configurable constant value.  I.e., a constant which may be initialised at program start.
```
Or something... 🙂

Reply:
Actually, I don't think this is used?  You wrap a `ValueDatum::Configurable` around a `Constant` and don't seem to need this wrapper?

Reply:
True. Remnant of an earlier attempt. Removed.